drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -62,6 +62,19 @@
 #define PLLSRC_FREQ 0
 #endif
 
+#define VCO_FREQ	(PLLSRC_FREQ / CONFIG_CLOCK_STM32_PLL_M_DIVISOR)
+#if (1000000UL <= VCO_FREQ && VCO_FREQ <= 2000000UL)
```

Comment:
Nit: Break a line here for readability:
```suggestion
#define VCO_FREQ	(PLLSRC_FREQ / CONFIG_CLOCK_STM32_PLL_M_DIVISOR)

#if (1000000UL <= VCO_FREQ && VCO_FREQ <= 2000000UL)
```

Reply:
Now it it more readable I can see it ;-)